### ISSUE TITLE
Problems with unicode characters

### ISSUE THREAD

**Issue by ghernadi** [SNIA/Swordfish-API-Emulator#29]
Python version: 2.7.12
The following list of index.json files contain the copyright symbol © (U+00A9), which cannot be parsed with the mentioned python version and causing an error when navigating to the corresponding sites.
```
Resources/StorageServices/1/StorageGroups/1/Volumes/Index.json
Resources/StorageServices/1/StorageGroups/2/Volumes/Index.json
Resources/StorageServices/1/StorageGroups/3/Volumes/Index.json
```

Additionally the following files start with a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) which also causes an error when navigating the corresponding sites:
```
Resources/StorageServices/1/ClientEndpointGroups/CEPG1/index.json
Resources/StorageServices/1/ClientEndpointGroups/CEPG1/EPC1/index.json
Resources/StorageServices/1/ServerEndpointGroups/SEPG1/EPC1/index.json
```

I'm not an expert with python, but I heard that there might be other solutions besides changing the json files.

**Comment by rahlvers** [comment]
@ghernadi - Thanks for logging the issue.  

A few comments:  
1.  Can you populate your Github profile with additional information?  Have you signed a SNIA CLA agreement?  As noted in the README for this repository, "Contributors that are not SNIA members must first agree to the terms of the SNIA Contributor Agreement for Non-Members: www.snia.org/cla".  We require the ability to correlate Github profiles with identifiable contributor information.

2.  This looks to be an error specifically in mockups.  We will take a look at whether these are current.

**Comment by ddeel** [comment]
This is still pending, and an appropriate fix will be rolled into a different pull request where agreeing to the SNIA CLA is not an open question.

**Comment by ddeel** [comment]
This Issue is addressed by pull request #45 .
